appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/model/media/SchemaImpl.java

```diff
@@ -354,6 +357,7 @@ public void removeProperty(String key) {
         this.properties.remove(key);
     }
 
+    @JsonProperty("additionalProperties")
```

Comment:
Isn't this the default name?

Reply:
Yes, I think I forgot to remove this after trying some things with jackson. Should be possible to remove.

Reply:
We can remove the explicit name, but the annotation is actually required to include the property. I think I found the reason back when I added it but can't remember. Anyhow, I remove the name but keep the annotation.